### PR TITLE
[docs] EAS Build: add section about support for multi-target ios projects

### PR DESCRIPTION
**Pull request by dsokal**
# Why

https://www.notion.so/expo/Multi-Target-Support-V1-Credentials-json-8d1c1c934f644ba6b098e33b456ffad8

To describe how to configure credentials.json to build a multi-target iOS project.

# How

Added a section in `Using local credentials`.

# Test Plan

I ran `yarn dev`, visited the local page, and made sure it renders correctly.

### CODE REVIEW

**Review by brentvatne — APPROVED** [review]

**Comment by brentvatne** [comment]
we should also update the ["Only single target iOS apps are supported"](https://docs.expo.io/build-reference/limitations/) section in limitations to clarify that it only applies to automatically managed credentials and that you can do this through local credentials now

**Comment by dsokal** [comment]
@brentvatne good point! I updated that page. Please take a look again, thanks!

**Review by wkozyra95 — APPROVED** [review]

**Review by brentvatne — APPROVED** [review]
looks great!

File: docs/pages/app-signing/local-credentials.md [hunk 1]
```diff
@@ -110,9 +110,42 @@ Create (or edit) `credentials.json` and configure it with the credentials:
...
+      "distributionCertificate": {
+        "path": "ios/certs/dist.p12",
+        "password": "DISTRIBUTION_CERTIFICATE_PASSWORD"
+      }
+    },
+    "shareextension": {
+      "provisioningProfilePath": "ios/certs/shareextension-profile.mobileprovision",
+      "distributionCertificate": {
+        "path": "ios/certs/another-dist.p12",
+        "password": "ANOTHER_DISTRIBUTION_CERTIFICATE_PASSWORD"
```
Comment (wkozyra95) [inline docs/pages/app-signing/local-credentials.md:139]: this suggests that you can't use the same dist cert

**Comment by erisvaldojunior** [comment]
Is it already possible to have multi-target credentials without having a local credentials.json?

**Comment by wkozyra95** [comment]
> Is it already possible to have multi-target credentials without having a local credentials.json?

yes, eas-cli will detect all targets that depend on the application target and will generate credentials for each of them

**Pull request merged**